app/routes.tsx

```diff
@@ -192,7 +207,7 @@ export const routes = createRoutesFromElements(
         </Route>
       </Route>
 
-      <Route index element={<Navigate to={pb.projects()} replace />} />
+      <Route index loader={() => redirect(pb.projects())} />
```

Comment:
Without this, when loading the root (an extremely common thing to do) you'd see the skeleton flash and then it would go blank wile the loaders for the next route load. I think doing it this way is fine as long we never link to `/`, which we don't. We'll have to see if there's any weird behavior, though. I think another workaround would be to leave the element thing in place and give this route a loader that just calls the loader from the projects page, that way the loading time is spent with the fallback up.